tests/testthat/test-parser.R

```diff
@@ -0,0 +1,44 @@
+test_that("Using Perceval version 0.12.24", {
+  tools_path <- "../../tools.yml"
+  tool <- yaml::read_yaml(tools_path)
+  perceval_path <- tool[["perceval"]]
+  perceval_version <- system2(perceval_path, args="--version", stdout=TRUE, stderr=FALSE)
+  expect_equal(perceval_version, "perceval 0.12.24")
+})
+test_that("Correct perceval path", {
+  tools_path <- "../../tools.yml"
+  tool <- yaml::read_yaml(tools_path)
+  perceval_path <- tool[["perceval"]]
+  expect_equal(file.exists(perceval_path), TRUE)
+})
+test_that("Correct mbox path", {
+  conf_path <-"../../conf/thrift.yml"
+  conf <- yaml::read_yaml(conf_path)
+  mbox_path <- conf[["mailing_list"]][["mbox"]]
+  expect_equal(file.exists(mbox_path), TRUE)
+})
```

Comment:
What this is really testing is that the data is available. This test will always fail. We can only test for what is specified on the repo. The third party tools is the borderline case of this type of unit tests. In that case, it is fine the test fails as this serves as a reminder there are third party tools the user has to download, on a specific version, and ideally in a specific place. User can then try to follow the README, then re-run unit tests to see if they pass. 

This makes sense and saves us time, because there are only a few tools needed. For data, not so much.

Reply:
Let's use this instead:

```r
test_that("configuration files are placed on recommended path ", {
  conf_path <-"../../conf/thrift.yml"
  expect_equal(file.exists(conf_path), TRUE)
})
```